infra/base-images/base-builder/jcc/jcc.go

```diff
@@ -357,6 +357,9 @@ func WriteStdErrOut(outstr string, errstr string) {
 	// Prints |outstr| to stdout, prints |errstr| to stderr, and saves |errstr| to err.log.
 	fmt.Print(outstr)
 	fmt.Fprint(os.Stderr, errstr)
+	// Record what source file produced the error.
+	// TODO: Should we write the actual flags used instead?
```

Comment:
Yes, I presume more info is better than less?

Reply:
[1e4e0e7](https://github.com/google/oss-fuzz/pull/11842/commits/1e4e0e7453c4713f588036409fe3e4c36153a7c9) Somewhat don't like this change as these functions are now doing multiple things together :(

Reply:
IIUC, you don't like it because it adds a lot of `return bin, args` to existing functions so that `WriteStdErrOut` can log them into `err.log`.
Would it help if we call `WriteStdErrOut()` in those functions instead?

Reply:
`WriteStdErrOut()` in `TryFixCCompilation()` changes the order of output and/or makes `err.log` more complicated. [Here](https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/jcc/jcc.go#L418) we print original err first, then fix err.
For the purpose of differentiating multiple JCC executions in err.log, writing `os.Args` probably works better for now.